authorize/evaluator/opa/policy/headers.rego

```diff
@@ -224,10 +224,10 @@ identity_headers := {key: values |
 
 	some i
 	[key, v1] := h[i]
-	values := [ v2 |
-	    some j
-	    [k2, v2] := h[j]
-	    key == k2
+	values := [v2 |
+		some j
```

Comment:
unrelated changes

Reply:
the original block was a mix of tabs & spaces formatting. We customary fix formatting issues part of other PRs.